### PR TITLE
Aerogear 7869 embed playground

### PR DESCRIPTION
**Pull request by secondsun**
## Motivation

This is the wiring for loading the playground react component.  

## What

This has the playground react component and it is hardwired to the localhosts graphql.  You can change this value by setting the endpoint in QueriesContainer.js

## Verification Steps
Make sure you are dunning the memeolist data-sync server on localhost:8080.
run this project (npm run server)
navigate to localhost:8080
navigate to Query
play!
## Checklist:

- [ ] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Progress

- [x] Mount tab inside UI
- [x] Test tab with server side by copying URL

## Additional Notes

screenshot from 2018-09-07 11-29-12

### CODE REVIEW

**Comment by wtrocki** [comment]
@secondsun Tests are failing. 

 I would say it looks aligned with the patternfly style outside the box, but I would rely on more experienced UI experts.

**Comment by secondsun** [comment]
@wtrocki I fixed a couple of small things.  Playground has some graphical issues out of the box, but they clear themselves up if you use it more.

For instance if you start typing a query it will be mixed in with the line numbers, but if you switch tabs it works from there on out.

Also playground was including a second graphql dependency and I've fixed that.

File: ui/components/queries/QueriesContainer.js [hunk 1]
```diff
@@ -1,5 +1,10 @@
 import React from "react";
+import { Provider } from "react-redux";
+import { Playground, store } from "graphql-playground-react";
 
-const QueriesContainer = () => <div>TODO</div>;
+const QueriesContainer = () => (
+    <Provider store={store}>
+        <Playground endpoint="http://localhost:8000/graphql" settings={{ "editor.theme": "light", "editor.cursorShape": "line" }} />
+    </Provider>);
```
Comment (josemigallas) [inline ui/components/queries/QueriesContainer.js]: Closing parenthesis goes in a new line.

File: ui/components/queries/QueriesContainer.js [hunk 2]
```diff
@@ -1,5 +1,10 @@
 import React from "react";
+import { Provider } from "react-redux";
+import { Playground, store } from "graphql-playground-react";
 
-const QueriesContainer = () => <div>TODO</div>;
+const QueriesContainer = () => (
+    <Provider store={store}>
+        <Playground endpoint="http://localhost:8000/graphql" settings={{ "editor.theme": "light", "editor.cursorShape": "line" }} />
```
Comment (josemigallas) [inline ui/components/queries/QueriesContainer.js]: I'd consider defining `settings` in a separate const.

**Pull request merged**